### ISSUE TITLE
Render amenity = monastery

### ISSUE THREAD

**Issue by geowas-github** [gravitystorm/openstreetmap-carto#3036]
Monasteries (amenity = monasteriy) are currently not rendered at all: https://www.openstreetmap.org/way/555465696
![grafik](https://user-images.githubusercontent.com/2790128/35412610-fa66c288-021c-11e8-9992-78d780b5e258.png)

To show a name, people tag monasteries as 
* tourism=attraction
(this is probably not their main purpose why they exist)
* place=locality https://www.openstreetmap.org/node/2701123706#map=19/28.55692/33.97485
* amenity=place_of_worship https://www.openstreetmap.org/relation/7924311
(this would be the churches, temples, etc. within the monastery)
All of these are not ideal in my opinion.

As monasteries play an important role in many religions they should be rendered. For me the name is enough. The area doesn't need to be highlighted necessarily, as it is should be covered with a lot of other things anyway:
* buildings (churches/temples/etc, accomodation, barns, ...)
* landuse (religious (I use it for the "core monastery"), agricultural land, etc.)
* parks
* other amenities (car parking, football pitch, ...)

If an area, the name could be rendered like other area labels in italics; colour like for place_of_worship.
Example: https://www.openstreetmap.org/way/555465696

If a node, render it like a place of worship.
Example: https://www.openstreetmap.org/node/2177650098

If a building, render it like a place of worship.
Example: https://www.openstreetmap.org/way/237666065

**Comment by kocio-pl** [comment]
What about landuse=religious?

**Comment by matkoniecz** [comment]
Yes, can you give case where landuse is tagged and amenity=monastery still needs a separate label?

**Comment by dieterdreist** [comment]
sent from a phone

> On 26. Jan 2018, at 06:18, Mateusz Konieczny <notifications@github.com> wrote:
> 
> Yes, can you give case where landuse is tagged and amenity=monastery still needs a separate label?


IMHO monasteries are interesting features of their own, regardless of landuse. Landuse=religious is not working well (both are properties and not features, “religion=*” already says something is religious, and landuse is a property as well).

**Comment by geowas-github** [comment]
Look at the example above: Mor Gabriel Monastery in SE Turkey:
https://www.openstreetmap.org/way/555465696
Everything within the ring wall is the monastery.
In the centre a church (amenity=place_of_worship) and the other main buildings around it (accomodation, refectory, ...). The area between these building in my opinion is the part which is the "religious" land (landuse = religious).
Around it we have farmland and grazing land for the cattle (everything north).

http://eichinger.ch/eichifamilyhom/Reisen/Tuerkei/Pfarreireise/MorGabriel/KlosterMorGabriel.jpg
https://en.wikipedia.org/wiki/Mor_Gabriel_Monastery

**Comment by polarbearing** [comment]
If we make progress with the outlines, this might be a solution for the monastery as well.

**Comment by kocio-pl** [comment]
I was first! :smile: We just need somebody to code it.

https://taginfo.openstreetmap.org/tags/amenity=monastery

**Comment by polarbearing** [comment]
I'm still sceptical to use a landuse colour for it. A monastery can be quite large and comprise different landuses, including farmland. I'd prefer an outline.

**Comment by kocio-pl** [comment]
Smaller landuses will be visible then.

**Comment by polarbearing** [comment]
The smaller landuses yes, but not the boundary. Imagine a monastery where the core buildings are surrounded by farmland, a typical constellation.

**Comment by matkoniecz** [comment]
Depends on whatever it is a correct representation of reality. In many cases it is - for example it may be

- field in monastery - landuse=farmland in landuse=religious
- small forest in landuse=industrial/religious/etc: landuse=forest in landuse=*
- construction site in the industrial complex - landuse=construction in landuse=industrial

it is mostly result of landuse values not excluding each other

**Comment by polarbearing** [comment]
Yes and in this particular case it is smaller landuses within a larger amenity=monastery

**Comment by Adamant36** [comment]
Thanks for the explanation. Going by that it seems that the smaller landuses inside the monastery might not be an issue. Or, maybe im wrong?

**Comment by polarbearing** [comment]
As said, you would not see where the monastery ends when the core buildings are surrounded e.g. by gaplessly mapped farmland.

**Comment by dieterdreist** [comment]
Am Sa., 24. Nov. 2018 um 21:29 Uhr schrieb polarbearing <
notifications@github.com>:

> The smaller landuses yes, but not the boundary. Imagine a monastery where
> the core buildings are surrounded by farmland, a typical constellation.
>


I would not expect the farmland as part of amenity=monastery, it would IMHO
be a tagging error

**Comment by Adamant36** [comment]
@dieterdreist, the wiki says 

> Monastery denotes the site of a monastery (or canonry, convent, comandry or hermitage), sometimes just a building, or a complex of buildings, that houses a room reserved for prayer (e.g. an oratory) as well as the domestic quarters and workplaces **(including gardens)**.

You think the tag should only be applied to the main building or something instead?

**Comment by CRCulver** [comment]
OSMAnd now renders amenity=monastery, which may spark much wider usage of this already used tag. I myself have begun tagging monasteries in the regions I map as such.

**Comment by dieterdreist** [comment]
> OSMAnd now renders amenity=monastery, which may spark much wider usage of this already used tag.
> 


great, thank you!

**Comment by jeisenbe** [comment]
Any other thoughts on this? Is it reasonable to use the same color for `landuse=religious`, `amenity=monastery`, and `amenity=place_of_worship` areas?

**Comment by matkoniecz** [comment]
`amenity=monastery` either is within `landuse=religious` or implies it.

Should we expect people to add `landuse=religious` to all `amenity=monastery`? If not, then rendering it like `landuse=religious` makes sense.

**Comment by jeisenbe** [comment]
Currently there are only 324 combinations of landuse=religious +
amenity=monastery on ways and relations, out of 4479 features (2714 of
which are ways or relations) tagged amenity=monastery.
(http://overpass-turbo.eu/s/PP5)

The wiki documentation does not mention this combination:
https://wiki.openstreetmap.org/wiki/Tag%3Aamenity%3Dmonastery

**Comment by polarbearing** [comment]
Looking back into the thread here, it is more an outline candidate, as it can comprise quite different landuses.

**Comment by dieterdreist** [comment]
sent from a phone

> On 17. Jan 2020, at 07:00, Joseph E <notifications@github.com> wrote:
> 
> Any other thoughts on this? Is it reasonable to use the same color for landuse=religious, amenity=monastery, and amenity=place_of_worship areas?


I would see monasteries as similar to churches (could be rendered the same or similar) while I never completely understood landuse=religious I would expect it to be less prominent in general

**Comment by polarbearing** [comment]
A monastery is often a larger areal than can comprise a church, residential buildings and agricultural landuse.

**Comment by dieterdreist** [comment]
Am Sa., 18. Jan. 2020 um 14:41 Uhr schrieb polarbearing <
notifications@github.com>:

> A monastery is often a larger areal than can comprise a church,
> residential buildings and agricultural landuse.
>


according to how you read it, also churches can comprise residential
buildings and agricultural landuse. The amenity=monastery tag is intended
for the site of an active monastery, and should not contain fields operated
by the monastery (garden yes, fields no).

**Comment by geowas-github** [comment]
> according to how you read it, also churches can comprise residential buildings and agricultural landuse. The amenity=monastery tag is intended for the site of an active monastery, and should not contain fields operated by the monastery (garden yes, fields no).

Of course it's all a matter of definition, and often a case by case thing.

But look at my initial example above and also compare to aerial imagery here: https://tools.geofabrik.de/mc/#16/37.3235/41.5369&num=2&mt0=mapnik&mt1=here-satellite
Mor Gabriel is one of the oldest christian monasteries and still active. There is not much around it. Being located in a nowadays not so much christian region - and from what I remember from my visit in 2001, rather hostile surrounding as I was told - it is a place of retreat if needed.

So looking at the pictures you can identify three possible "borders" for this monastery. 
red - the core piece, church and residential buildings
blue - plus gardens, orchards, agricultural buildings, soccer field, etc. (as seen here: http://eichinger.ch/eichifamilyhom/Reisen/Tuerkei/Pfarreireise/MorGabriel/KlosterMorGabriel.jpg)
orange - plus grazing area for livestock and a basin for water supply.

![morgabriel](https://user-images.githubusercontent.com/2790128/73137764-29c27480-405c-11ea-8ca3-31e52b2833bd.png)

In this case I tagged the complete orange area a monastery, because even that bigger area is enclosed by a big wall, clearly delimiting the inside from the definitively-not-monastery area at the outside.

Eeven if we come to the conclusion that blue would be the "right" extent of the monastery then it will comprise landuses other than religious (I think the "red" monastary will not be able to survive). And this should preferably be reflected in the rendering decision.

**Comment by dieterdreist** [comment]
yes, I think it will represent either the orange or the blue area, at
discretion of the mapper / according to context (fencing / walls, if any,
etc.).

**Comment by ppete2** [comment]
I support rendering Monasteries (_amenity=monastery_). They are important, well-know amenities. Often also promoted and open for touristic visits. So it's definately worth to see their location on a map.

How to render? As some of you already stated, monasteries areas are usually of a somewhat extensive size, including several landuses. The religious place-of-worship is often just the monastery's church building. But there can be lots of other things belonging to a monasteries' area like: buildings for accommodation of monks and guests, parks, garden, little shops, museums or restaurants and even schools withing the walls of a monastery. So I would not render this amenity with a dark "religious" colour but mayeb with same grey as used for _"landuse=residential"_ . If at all: Most important would be at least a label with the monasteries name in the center of its area.

If grazing areas, forests and farmland which are operated by the monastery and still within some kind of walls i.e. belonging to the "extended area of the monterstery" should be part of _amenity=monastery_ is question of tagging and not rendering. And since _amenity=monastery_ is not a landuse, these areas should be tagged with their corresponding landuses anyway (forest, meadow, farmland)

**Comment by imagico** [comment]
I would be open to rendering amenity=monastery with a point label or icon + label.  I see fairly little indication that there is a good basis for rendering those with a polygon fill.  Most use of the tag is either on nodes or on polygons also tagged as a buildings and many of the other polygons seem rough hull sketches not representing a verifiable extent of the monastery.  And since we render names on buildings we currently incentivize tagging this on a building rather than on the whole monastery.

Designing a suitable symbol that is religion neutral and intuitive to understand, preferably with the possibility to differentiate by religion where tagged, is of course difficult.

Dominant religions in practical use are buddhist and christian by the way.

**Comment by HolgerJeromin** [comment]
We could try `@amenity-brown` (which is not so dark/prominent as place_of_worship black: `#734a08`) with the corresponding religious icons (same as place_of_worship).
Or we could have the generic  `place_of_worship.svg` for all of them.

**Comment by imagico** [comment]
That is not in any way intuitive - how is a monastery in any way more of an amenity than a place of worship?  And we don't want to communicate to mappers that place_of_worship and monastery is all the same to us and that they can use the tags interchangeably.

I suggest to people who want this rendered to put some effort and ambition into developing a well suitable, distinctive and intuitive symbolization.

**Comment by crse** [comment]
`amenity=monastery` usage seems to explode in the past one year. From 8800 in June 2024 to 13800 as 30 April 2025.
https://taghistory.raifer.tech/#***/amenity/monastery

Would be cool if the render can be implemented ASAP :)

**Comment by imagico** [comment]
That 'explosion' is located in Myanmar by the way:

https://taginfo.geofabrik.de/asia:myanmar/tags/amenity=monastery#chronology

**Comment by dch0ph** [comment]
> Would be cool if the render can be implemented ASAP :)

Do you have an icon proposal? That is what somebody needs to develop :)
